core/controlplane/config/config.go

```diff
@@ -113,6 +113,10 @@ func NewDefaultCluster() *Cluster {
 			MapPublicIPs:       true,
 			Experimental:       experimental,
 			ManageCertificates: true,
+			AmazonSsmAgent: AmazonSsmAgent{
+				Enabled:     false,
+				DownloadUrl: "https://github.com/DailyHotel/amazon-ssm-agent/releases/download/v2.0.805.1/ssm.linux-amd64.tar.gz",
```

Comment:
This URL looks suspicious and doesn't seem to be needed in the code, as it does also appear in the configuration. I rather not have a binary on my system that wants to download from "somewhat arbitrary" github places by default for configuring my production environments.

Would it not be possible to set the default to an empty string, and make the user explicitly acknowledge the download location by having to uncomment it in the configuration file?